### ISSUE TITLE
String-boundary check not working for string fields in response schemas

### ISSUE THREAD

**Issue by shashank90** [IBM/openapi-validator#400]
```
components:
  responses:
    BadRequest:
      description: Bad request
      content:
        application/json:
          schema:
            type: object
            required:
            - error
            properties:
              error:
                type: object
                required:
                - code
                - message
                properties:
                  code:
                    type: string
                    pattern: ^w+$
                  message:
                    type: string
                    pattern: ^w+$
```
lint-openapi isn't catching missing string-boundary checks from response schemas. In the above example,  code & message are without minLength & maxLength.

I am using version: 0.57.4

Am I missing something?

**Comment by dpopp07** [comment]
@shashank90 Yes, this is intentional. We only validate string boundaries for "input" - parameters and request body properties. See [the `given` property in the rule definition](https://github.com/IBM/openapi-validator/blob/main/packages/ruleset/src/rules/string-boundary.js#L10-L16).
